core/tests/webdriverio_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -133,6 +133,8 @@ var ExplorationEditorFeedbackTab = function() {
   };
 
   this.selectLatestFeedbackThread = async function() {
+    await waitFor.visibilityOf(
+      suggestionRowElement, 'Suggestion row is taking too long to appear');
     var suggestionRowFirst = await $$(suggestionRowClassName)[0];
```

Comment:
We need to wait for the element to be visible otherwise `$$` was returning undefined and this was causing this flake: https://github.com/oppia/oppia/actions/runs/3173358543/jobs/5168885590